src/main/java/com/faforever/client/fx/DualStringListCellController.java

```diff
@@ -21,6 +27,19 @@ public void setLeftText(String apply) {
     left.setText(apply);
   }
 
+  public void setLeftTextTooltip(String apply) {
+    final WebView web = new WebView();
+    uiService.registerWebView(web);
+    web.setPrefSize(web.getPrefWidth() / 2, web.getPrefHeight() / 2);
+    web.getEngine().loadContent(apply);
+    final Tooltip  tip = new Tooltip();
+    tip.setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+    tip.setGraphic(web);
+    tip.setShowDelay(Duration.ZERO);
+    tip.setHideDelay(Duration.seconds(3));
+    left.setTooltip(tip);
```

Comment:
we try to stick to mvc . So it should be done in fxml file. Either in this one for a serpate one. And further DualStringListCellController is a Class that is potentially used else where as well adding this pretty sepecific behaviour here seems strange, especially when the method is called setLeftTextTooltip, where it is rather addTooltipWithWebview. So yes u might add this webview capabilities here but make sue your naming reflects that it is not a normal tooltip. Also try define Webview and maybe even tooltip in fxml. On the other hand if it is hard to define in FXML I guess we can leave it tho. Further 3 seconds seem to be short. And dividing the pref size by half is also strange. Why do u do it? Maybe just hard code it :D Could even make use units dpending on screen size.